packages/babel-preset/README.md

```diff
@@ -134,3 +134,131 @@ As noted above, you can include multiple of these presets together. Some common
   }
 }
 ```
+
+## Experimental Presets
+
+## `@shopify/babel-preset/common`
+The `@shopify/babel-preset/common` preset is designed to be a common import that offers more options. It is a combination of the `web`, `node`, and `react` presets all in one.
+
+### Usage
+
+```javascript
+{
+  "babel": {
+    "presets": [
+      ["@shopify/babel-preset/common", {"typescript": true}]
+    ]
+  }
+}
+```
+
+### Options
+
+#### `debug`
+
+`boolean`, defaults to `false`.
+
+Outputs to console.log the polyfills and transform plugins enabled by preset-env and, if applicable, which one of your targets that needed it.
+
+[Documentation](https://babeljs.io/docs/en/babel-preset-env#debug)
+
+
+#### `modules`
+
+`"amd" | "umd" | "systemjs" | "commonjs" | "cjs" | "auto" | false`, defaults to `"auto"`.
+
+Enable transformation of ES module syntax to another module type. Note that `cjs` is just an alias for `commonjs`.
+
+Setting this to `false` will preserve ES modules. Use this only if you intend to ship native ES Modules to browsers. If you are using a bundler with Babel, the default `modules: "auto"` is always preferred.
+
+##### `modules: "auto"`
+
+By default `@babel/preset-env` uses [`caller`](options.md#caller) data to determine whether ES modules and module features (e.g. `import()`) should be transformed. Generally `caller` data will be specified in the bundler plugins (e.g. `babel-loader`, `@rollup/plugin-babel`) and thus it is not recommended to pass `caller` data yourself -- The passed `caller` may overwrite the one from bundler plugins and in the future you may get suboptimal results if bundlers supports new module features.
+
+[Documentation](https://babeljs.io/docs/en/babel-preset-env#modules)
+
+#### `typescript`
+Enables `@babel/preset-typescript` to transfrom TypeScript into Javascript.
+
+[Documentation](https://babeljs.io/docs/en/babel-preset-typescript)
+
+#### `useBuiltIns`
+
+`"usage"` | `"entry"` | `false`, defaults to `false`.
+
+This option configures how `@babel/preset-env` handles polyfills.
+
+When either the `usage` or `entry` options are used, `@babel/preset-env` will add direct references to `core-js` modules as bare imports (or requires). This means `core-js` will be resolved relative to the file itself and needs to be accessible.
+
+[Documentation](https://babeljs.io/docs/en/babel-preset-env#usebuiltins)
+
+#### `transformRuntime`
+
+`boolean`, defaults to `false`
+
+This option when `true` will enable the `@babel/plugin-transform-runtime` plugin.
+
+[Documentation](https://babeljs.io/docs/en/babel-plugin-transform-runtime)
+
+#### `transformRuntimeOptions`
+
+This option configures the `@babel/plugin-transform-runtime` plugin when the `transformRuntime` option has been enabled.
```

Comment:
Do we need to explain each of these option in our own words? Could we get away with linking to https://babeljs.io/docs/en/babel-plugin-transform-runtime#options and noting any cases where our defaults differ from babel.

Same goes for the reactPresetOptions too.

There's probably something we can do with the other options to say "same as preset-env" too.